content/ux-strat-bootcamp/week-one/tuesday/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Tuesday
+ready: true
+---
+
+# **Mini Design Challenge:**
+
+## Design a Better Commute
+
```

Comment:
Hmm. This ontent doesn't make a lot of sense. What is the applicant supposed to do?